packages/plugins/max-aliases/src/index.ts

```diff
@@ -78,8 +78,8 @@ class MaxAliasesVisitor {
     if (
       'alias' in node &&
       node.alias &&
-      node.name.value !== '__typename' &&
-      !this.config.allowList.includes(node.alias.value)
+      !this.config.allowList.includes(node.alias.value) &&
+      !this.config.allowList.includes(node.name.value)
```

Comment:
From this change it looks like the allow list just does not work and that you would fix it here too, is that the case ? If so many thanks for the fix :)

Reply:
Hi @LMaxence! This condition is determining whether a node in GraphQL query should count towards the number of aliases. Previously it explicitly excluded `__typename`, but this change ensures that it only includes nodes _not_ in the `allowList`. My understanding is that you need to check both `node.name` and `node.alias` to make sure it works, and I confirmed this with the tests.